matter_server/server/device_controller.py

```diff
@@ -584,15 +586,25 @@ async def remove_node(self, node_id: int) -> None:
         )
         fabric_index = node.attributes[attribute_path]
 
-        self.server.signal_event(EventType.NODE_REMOVED, node_id)
-
-        await self.chip_controller.SendCommand(
-            nodeid=node_id,
-            endpoint=0,
-            payload=Clusters.OperationalCredentials.Commands.RemoveFabric(
-                fabricIndex=fabric_index,
-            ),
+        result: Clusters.OperationalCredentials.Commands.NOCResponse = (
+            await self.chip_controller.SendCommand(
+                nodeid=node_id,
+                endpoint=0,
+                payload=Clusters.OperationalCredentials.Commands.RemoveFabric(
+                    fabricIndex=fabric_index,
+                ),
+            )
         )
+        if (
+            result.statusCode
+            == Clusters.OperationalCredentials.Enums.NodeOperationalCertStatusEnum.kOk
+        ):
+            LOGGER.info("Successfully removed Home Assistant fabric from device.")
+        else:
+            LOGGER.warning(
+                "Removing Home Assistant fabric from device failed with status code %d.",
```

Comment:
strictly taken this is not the home assistant fabric but the fabric from the consumer of our lib.

Reply:
Yeah good point :+1:  Thanks!